### PR TITLE
Reviewed application-resources-monitoring-zh.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu patrickluo@yunify.com
/assign @Sherlock113

### CODE REVIEW

**Comment by Patrick-LuoYu** [comment]
Relocated images.
/assign @Sherlock113

File: content/zh/docs/cluster-administration/application-resources-monitoring.md [hunk 1]
```diff
@@ -1,35 +1,32 @@
...
-description: "Monitor application resources in KubeSphere"
+keywords: "Kubernetes, KubeSphere, 资源, 监控"
+description: "在 KubeSphere 监控应用资源。"
 linkTitle: "应用资源监控"
 weight: 8300
 ---
 
 
-除了在物理资源级别监控数据外，集群管理员还需要密切跟踪整个平台上的应用资源，例如DevOps项目的数量，以及特定类型的工作负载和服务的数量。应用资源监控提供了平台的资源使用情况和应用程序级别趋势的摘要。
+除了在物理资源级别监控数据外，集群管理员还需要密切跟踪整个平台上的应用资源，例如项目和 DevOps 项目的数量，以及特定类型的工作负载和服务的数量。**应用资源监控**提供了平台的资源使用情况和应用级趋势的汇总信息。
```
Comment (Sherlock113) [inline content/zh/docs/cluster-administration/application-resources-monitoring.md:10]: DevOps 项目
->
DevOps 工程

File: content/zh/docs/cluster-administration/application-resources-monitoring.md [hunk 2]
```diff
@@ -1,35 +1,32 @@
...
-![Time Range](/images/docs/cluster-administration/application-resources-monitoring-zh/time-range.png)
-
-5. 单击特定资源以查看特定时间段内的详细使用情况和趋势，例如**集群资源使用情况**下的**CPU**。详细信息页面允许您按项目查看特定的监控数据。 高度交互的仪表板使用户可以自定义时间范围，显示给定时间点的确切资源使用情况。
-![Cluster Resources Usage](/images/docs/cluster-administration/application-resources-monitoring-zh/cluster-resources-monitoring.png)
+1. 点击左上角的**平台管理**，然后选择**集群管理**。
+![Platform](/images/docs/zh-cn/cluster-administration/application-resources-monitoring/platform.png)
+2. 如果您已启用了[多集群功能](../../multicluster-management
+)并已导入了 Member 集群，您可以选择一个集群以查看其应用程序资源。如果尚未启用该功能，请直接进行下一步。
+![Clusters Management](/images/docs/zh-cn/cluster-administration/application-resources-monitoring/clusters-management.png)
+3. 在左侧导航栏选择**监控告警**下的**应用资源**以查看应用资源概览，包括群集中所有资源使用情况的汇总信息。
```
Comment (Sherlock113) [inline content/zh/docs/cluster-administration/application-resources-monitoring.md:23]: 群集
->
集群

**Review by Sherlock113 — COMMENTED** [review]
Have a look and my comments and make changes. Thanks.

**Comment by Patrick-LuoYu** [comment]
Fixed issues based on comments.
/assign @Sherlock113

**Pull request merged**